resources/assets/js/stories/JobPosterBuilder/Intro.stories.tsx

```diff
@@ -1,14 +1,47 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
 import { withIntl } from "storybook-addon-intl";
 import { action } from "@storybook/addon-actions";
-import JobBuilderIntro from "../../components/JobBuilderIntro/JobBuilderIntro";
+import fakeJob from "../../fakeData/fakeJob";
+import IntroForm from "../../components/JobBuilderIntro/IntroForm";
+import { Job } from "../../models/types";
 
-const stories = storiesOf("Job Poster Builder|Intro", module).addDecorator(
-  withIntl,
-);
+const stories = storiesOf("Job Poster Builder|Intro", module)
+  .addDecorator(withInfo)
```

Comment:
I've added the `withInfo` and `withKnobs` decorators to `config.ts` so we don't need to include them on every story individually